docs/Spectre.DataValidatorApp.md

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
divik2spectre.py is not mentioned here. What if users would like to use it? Is that script just for us?

Reply:
This script is intended for our internal purposes *only*.